tests/conftest.py

```diff
@@ -117,8 +118,8 @@ def csm(web3):
 # ---- Lido contracts ----
 @pytest.fixture()
 def contracts(web3, provider):
-    src.variables.LIDO_LOCATOR_ADDRESS = "0x548C1ED5C83Bdf19e567F4cd7Dd9AC4097088589"
-    src.variables.CSM_MODULE_ADDRESS = "0xdA7dE2ECdDfccC6c3AF10108Db212ACBBf9EA83F"
+    src.variables.LIDO_LOCATOR_ADDRESS = variables.LIDO_LOCATOR_ADDRESS
```

Comment:
This change is senseless, src.variables.LIDO_LOCATOR_ADDRESS == variables.LIDO_LOCATOR_ADDRESS, result will be the same if we just remove this line.

You need something like
```suggestion
if not src.variables.LIDO_LOCATOR_ADDRESS:
    src.variables.LIDO_LOCATOR_ADDRESS = "0x548C1ED5C83Bdf19e567F4cd7Dd9AC4097088589"
```
Otherwise this will break our tests on mainnet after merge

Reply:
fixed